web/pdf_attachment_viewer.js

```diff
@@ -116,13 +116,12 @@ class PDFAttachmentViewer {
   /**
    * @param {PDFAttachmentViewerRenderParameters} params
    */
-  render(params = {}) {
-    var attachments = params.attachments || null;
-    var attachmentsCount = 0;
+  render({ attachments, keepRenderedCapability = false, }) {
+    attachments = attachments || null;
```

Comment:
I thought about this while writing the patch, and ended up purposely leaving this code intact (given that default parameters are only applied when `param === undefined`).
Please note that the `attachments` parameter should *not* be regarded as optional, which is what a default parameter suggests!

In this case, what we pass to `render` in https://github.com/mozilla/pdf.js/blob/9e4dcf156b6d31926b8d48112ee6625b674b1ddc/web/app.js#L1046 comes directly from the API. Currently we'll return `null`, coming from https://github.com/mozilla/pdf.js/blob/9e4dcf156b6d31926b8d48112ee6625b674b1ddc/src/core/obj.js#L318, when there're no attachments present. But I figured that it makes sense for `web/pdf_attachment_viewer.js` to not assume that, and always fallback to `null` here.

Hence it might be most correct to actually do something like `attachments = isArray(attachments) ? attachments : null;` instead here!?